client.go

```diff
@@ -485,9 +614,9 @@ type dialResult struct {
 
 func countDialResult(err error) {
 	if err == nil {
-		torrent.Add("successful dials", 1)
+		torrentVar.Add("successful dials", 1)
```

Comment:
can likely clean most of this noise up by changing `type torrent` to `type download`